pkg/aws/pca_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/acmpca/types"
 	acmpcatypes "github.com/aws/aws-sdk-go-v2/service/acmpca/types"
 
-	testLog "github.com/go-logr/logr/testing"
+	testLog "github.com/go-logr/logr"
```

Comment:
Can we change this to `logr`? It's not the testing package from `logr` anymore, and that way it matches `logrtesting` from the other file

Reply:
Removed testLog